### PR TITLE
Sass updates

### PR DESCRIPTION
**Pull request by Himadu2000**

### CODE REVIEW

File: theme/assets/scss/account.scss [hunk 1]
```diff
@@ -240,14 +240,16 @@ tbody tr td:first-child {
...
 }
   
-/* for custom scrollbar for webkit browser*/  
+// for custom scrollbar for webkit browser
 ::-webkit-scrollbar {
     width: 1px;
 } 
 ::-webkit-scrollbar-track {
     -webkit-box-shadow: inset 0 0 1px rgba(0,0,0,0.3); 
+    box-shadow: inset;
```
Comment (capfsb) [inline theme/assets/scss/account.scss:249]: what it should do? 
-webkit-box-shadow == box-shadow
I think need only box-shadow

**Review by capfsb — APPROVED** [review]

**Review by vamcart — COMMENTED** [review]
I'm not understand.
What benefits of this changes?!

Just for stats, more PR's or what?!

**Comment by capfsb** [comment]
Himada removed deadcode, its clean up, i think we can merge when he fix box-shadow

File: theme/assets/scss/account.scss (hunk 1 above)
Comment (Himadu2000) [reply theme/assets/scss/account.scss:249]: > what it should do?
> -webkit-box-shadow == box-shadow
> I think need only box-shadow

Well it gives compatibility errors if both of them is not there so, feel free to do anything to fix that if you wish...

**Pull request merged**